controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -2101,9 +2101,6 @@ bool master_thread::handle_intel_slave_join(
     database.set_node_ipv4(radio_mac, bridge_ipv4);
     database.set_node_manufacturer(radio_mac, "Intel");
 
-    database.set_hostap_supported_channels(radio_mac, notification->hostap().supported_channels,
```

Comment:
@arnout already mentioned it, 
please remove supported_channels from sNodeHostap since this is the only place its used 

use get_hostap_supported_channels_string() instead of using this code  (line 2083)
```
        for (auto supported_channel : notification->hostap().supported_channels) {
            if (supported_channel.channel > 0) {
                LOG(DEBUG) << "supported_channel = " << int(supported_channel.channel);
            }
        }
```

Reply:
with regard to this comment and the one made by @arnout
The supported channels still need to be part of the sNodeHostap so they will be sent to the son_slave_thread as part of the `ACTION_APMANAGER_JOINED_NOTIFICATION` message

because of this when the radio capabilities are sent as part of the `cACTION_CONTROL_SLAVE_JOINED_NOTIFICATION` message and can be disregard

Reply:
> with regard to this comment and the one made by @arnout
> The supported channels still need to be part of the sNodeHostap so they will be sent to the son_slave_thread as part of the `ACTION_APMANAGER_JOINED_NOTIFICATION` message
> 
> because of this when the radio capabilities are sent as part of the `cACTION_CONTROL_SLAVE_JOINED_NOTIFICATION` message and can be disregard

basically you are right,  but since we added the `preferred_channels ` list and always update it ,
 see here:
ACS report - preffered_channels used
no ACS report - `update_preference_channels_from_supoorted_channels() `used

then, you  can remove supported_channels from the cACTION_CONTROL_SLAVE_JOINED_NOTIFICATION as well.

Another thing , 
notification->hostap().supported_channels is still used (line 2083 area)
use `get_hostap_supported_channels_string()` instead

Reply:
Moved the supported_channels to outside the sNodeHostap
And changed all previews missuses of `supported_channels` to `preferred_channels`